lib/gems/pending/util/miq-process.rb

```diff
@@ -127,21 +120,23 @@ def self.processInfo(pid = nil)
     result
   end
 
+  # Return the command line string for the given +pid+. If the pid has already
+  # exited, or there is some sort of permissions issue that causes it to be set
+  # to nil, then return an empty string instead.
+  #
   def self.command_line(pid)
     # Already exited pids, or permission errors cause ps or ps.cmdline to be nil,
     # so the best we can do is return an empty string.
     Sys::ProcTable.ps(:pid => pid).try(:cmdline) || ""
   end
 
+  # Return whether or not the given +pid+ is running.
+  #
   def self.alive?(pid)
-    raise NotImplementedError, "Method MiqProcess.alive? not implemented on this platform [#{Sys::Platform::IMPL}]" unless Sys::Platform::OS == :unix
-
-    begin
-      Process.kill(0, pid)
-      true
-    rescue Errno::ESRCH
-      false
-    end
+    Process.kill(0, pid)
```

Comment:
Any reason not to use https://github.com/ManageIQ/more_core_extensions/pull/73/files#diff-b07041a64a859612407de7d132fe52bdR47 ?

Reply:
@bdunne You want me to just yank it completely? I couldn't find anything using it currently. The miq_web_server_worker_mixin.rb was using it, but it seems the relevant code got deleted last year.

@NickLaMuro Am I missing any calls elsewhere?

Same for pause and resume probably, since the current code was windows-only and was broken.

Reply:
Those sound like two very different questions (use a gem or remove). I think @bdunne is suggesting an alias for short term compatibility rather than copying the method contents.

Reply:
@carbonin pretty sure there's no way the current code is being used. It's only written for Windows, and that code appears to be invalid.

@bdunne, @carbonin, @NickLaMuro I've removed it. Anything that needs it can use `more_core_extensions`.